src/reason-parser/reason_parser.mly

```diff
@@ -94,20 +84,20 @@ open Ast_mapper
    module Typ = struct
     val mk: ?loc:loc -> ?attrs:attrs -> core_type_desc -> core_type
     let mk ?(loc = !default_loc) ?(attrs = []) d =
-       {ptyp_desc = d; ptyp_loc = loc; ptyp_attributes = attrs}
+       {Parsetree.ptyp_desc = d; Parsetree.ptyp_loc = loc; ptyp_attributes = attrs}
      ..
    end
 
    parse_tree.mli
    --------------
    and core_type = {
-     ptyp_desc: core_type_desc;
-     ptyp_loc: Location.t;
+     Parsetree.ptyp_desc: core_type_desc;
```

Comment:
You only need to do this on one of the record labels. I notice you've done it twice in several cases.

Reply:
Want to make sure you just do it at most once per record? (It will help reduce merge conflicts).